src/main/java/org/openhab/binding/zwave/internal/converter/ZWaveDoorLockConverter.java

```diff
@@ -92,13 +100,24 @@ private State handleEventLockState(ZWaveThingChannel channel, ZWaveCommandClassV
     }
 
     private State handleEventCondition(ZWaveThingChannel channel, ZWaveCommandClassValueEvent event) {
-        if (!channel.getUID().getId().equals("sensor_door")) {
+        if (!channel.getUID().getId().startsWith("sensor_door")) {
```

Comment:
Why have you changed this? There are no other such channels with this name. Probably this should be using the channel type anyway.

Reply:
Yes I think that should be rolled back. It was a fix idea I do this morning, the other code was tested with my lock the whole weekend.

Reply:
type should be better.
Is this channel.getChannelTypeUID().getId()?
Then I will modify it...

Reply:
Yes, I think this should be best, but please try it to be sure :)

Reply:
Hi Chris, 
yes, I've tested. It is productive since Monday. 
Daniel